client/my-sites/plans-features-main/index.tsx

```diff
@@ -66,19 +66,19 @@ import isEligibleForWpComMonthlyPlan from 'calypso/state/selectors/is-eligible-f
 import { isUserEligibleForFreeHostingTrial } from 'calypso/state/selectors/is-user-eligible-for-free-hosting-trial';
 import { getCurrentPlan, isCurrentUserCurrentPlanOwner } from 'calypso/state/sites/plans/selectors';
 import { getSitePlanSlug, getSiteSlug, isCurrentPlanPaid } from 'calypso/state/sites/selectors';
+import usePlanIntentFromSiteMeta from '../hooks/use-check-plan-availability-for-purchase';
```

Comment:
I believe we meant to import from a different file because the dev build is failing. I conducted tests with the updates below, but if I'm mistaken feel free to let me know. Happy to retest 🙂

```suggestion
import usePlanIntentFromSiteMeta from './hooks/use-plan-intent-from-site-meta';
```

Reply:
thanks for spotting these! really weird testing on my side (did I even?) 🤔